src/test/HostActivationTests/GivenThatICareAboutMultilevelSharedFxLookup.cs

```diff
@@ -595,7 +615,12 @@ public void Ensure_On_NonWindows_Multiple_Global_Locations_Are_Probed()
             {
                 return;
             }
-            
+            else
+            {
+                // Currently no support for global locations on Linux; remaining test code below is preserved for now.
```

Comment:
I agree on not having 'dead code' but there was a fairly recent change (issue #1553) that disabled the Windows side, and I wasn't sure on the priority of fixing that, although it seems pretty high to me since we still depend on \Program Files type functionality for the non-developer (production) case. So I was assuming we will fix the Windows side soon.